libc/src/unistd/pathconf_utils.h

```diff
@@ -0,0 +1,24 @@
+//===-- Implementation header for pathconf_utils ----------------------*- C++
+//-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_UNISTD_PREAD_H
+#define LLVM_LIBC_SRC_UNISTD_PREAD_H
+
+#include <unistd.h>
+
+namespace LIBC_NAMESPACE {
+
+static long filesizebits(const struct statfs &s);
```

Comment:
USE `LIBC_INLINE` instead of `static`. Or remove static if you want to make it an object library

Reply:
if I add LIBC_INLINE to the function signature in the header file, I dont need it in the cpp file right?